packages/shared-ux/chrome/navigation/src/ui/v2/default_navigation.tsx

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { FC, useCallback, useState } from 'react';
+import { ChromeProjectNavigationNode } from '@kbn/core-chrome-browser';
+
+import { Navigation } from './components';
+
+interface Props {
+  navTree: ChromeProjectNavigationNode[];
+}
+
+export const DefaultNavigation: FC<Props> = ({ navTree }) => {
```

Comment:
I think the props are "public," so I would suggest giving them a clear name and a descriptive comment:
```suggestion
/**
 * @public
 */
export interface DefaultNavigationProps {
  /**
   * A navigation tree structure with object items containing labels, links, and sub-items
   */
  navTree: ChromeProjectNavigationNode[];
}

export const DefaultNavigation: FC<DefaultNavigationProps> = ({ navTree }) => {
```